tests/testthat/test-plot-grid.R

```diff
@@ -0,0 +1,52 @@
+test_that("plots grid is rendered correctly", {
+  skip_if_not_installed("vdiffr")
+  skip_if_not_installed("ggplot2")
+  skip_if_not_installed("tlf")
```

Comment:
is `skip_if_not_installed("ggplot2")` really required? because if tlf is installed then ggplot2 must be installed as well.

Reply:
Removed this line.